From b90fcdbe0405e5f3c46c967c55b4d3921c1b9a91 Mon Sep 17 00:00:00 2001
From: Leonardo Sandoval <leonardo.sandoval.gonzalez@linux.intel.com>
Date: Mon, 2 Sep 2019 09:33:30 -0500
Subject: [PATCH 26/26] return NULL after breaking while
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

fixes:

fftools/ffmpeg.c:2442:5: error: ‘return’ with no value, in function returning non-void [-Werror=return-type]
 2442 |     return;
      |     ^~~~~~
---
 fftools/ffmpeg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fftools/ffmpeg.c b/fftools/ffmpeg.c
index e8a32d1..a02adba 100644
--- a/fftools/ffmpeg.c
+++ b/fftools/ffmpeg.c
@@ -2439,7 +2439,7 @@ static void *filter_pipeline(void *arg)
             break;
         }
     }
-    return;
+    return NULL;
 }
 #endif
 static int send_frame_to_filters(InputStream *ist, AVFrame *decoded_frame)
-- 
2.23.0

